packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/MultiFrameViewer/FrameCarousel.js

```diff
@@ -0,0 +1,153 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+import locationValidator from '../../helpers/locationValidator'
+import getViewer from '../../helpers/getViewer'
```

Comment:
probably disregard as will be used subsequently, but `getViewer`, as well as grommet `List`, `Text` and `Icons` not used